File: calc.lua
print("Your first number:")
local num1 = io.read()
print("Your second number:")
local num2 = io.read()
print("Choose operator: + or - or * or /")
local operator = io.read()

if operator == "+" then
   local add = num1 + num2
   print(add)
end 
if operator == "-" then
   local subtract = num1 - num2
   print(subtract)
end
if operator == "*" then
   local multiply = num1 * num2
   print(multiply)
end
if operator == "/" then
   local divide = num1 / num2
   print(divide)
end
-- lua rating weird🤨/10

File: calc.ts
let num1 = window.prompt("enter your first number")!
let num2 = window.prompt("enter your second number")!
let sign = window.prompt("enter your sign")!
let sign1 = "+"
let sign2 = "-"
let sign3 = "*"
let sign4 = "/"

if (sign == sign1) {
    window.alert(~~num1+~~num2)
}
if (sign == sign2) {
    window.alert(~~num1-~~num2)
}
if (sign == sign3) {
    window.alert(~~num1*~~num2)
}
if (sign == sign4) {
    window.alert(~~num1/~~num2)
}